pandas/tests/computation/test_eval.py

```diff
@@ -1757,6 +1757,29 @@ def test_no_new_globals(self, engine, parser):
         assert gbls == gbls2
 
 
+class TestEvalNamedKWargs(object):
+    # xref https://github.com/pandas-dev/pandas/issues/25813
+
+    def test_eval_with_kwargs(self, engine, parser):
+        def fn(x):
+            return 2 * x
+        result = pd.eval("fn(x=2)", engine=engine, parser=parser)
+        assert result == 4
+
+    def test_query_str_contains(self, parser):
+        df = pd.DataFrame([["I", "XYZ"], ["IJ", None]], columns=['A', 'B'])
+
+        expected = df[df["A"].str.contains("J")]
+        result = df.query("A.str.contains('J')", engine="python",
```

Comment:
Any reason to not parametrize this for all engines?

Reply:
Yes, because `engine=pandas` fails for this test.  It's tries to hash a `Series` object.